python/pyspark/sql/readwriter.py

```diff
@@ -114,6 +114,14 @@ def option(self, key, value):
             * ``pathGlobFilter``: an optional glob pattern to only include files with paths matching
                 the pattern. The syntax follows org.apache.hadoop.fs.GlobFilter.
                 It does not change the behavior of partition discovery.
+            * ``modifiedBefore``: an optional timestamp to only include files with
+                modification times occurring before the specified time.  The provided timestamp
+                must be in the following format:  YYYY-MM-DDTHH:mm:ss
```

Comment:
Could you use a single space in `format:  YYYY-`?

Reply:
Completed throughout files.